zerver/lib/avatar_hash.py

```diff
@@ -21,16 +21,16 @@ def gravatar_hash(email):
     # not error out on it.
     return make_safe_digest(email.lower(), hashlib.md5)
 
-def user_avatar_hash(email):
+def user_avatar_hash(uid):
```

Comment:
Add a comment to this function that if is modified, we need to also modify migration 0060.